eclipse_plugin/.classpath

```diff
@@ -4,6 +4,6 @@
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry exported="true" kind="lib" path="lib/guava-19.0.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/google-java-format-1.1-SNAPSHOT.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/google-java-format-1.3-SNAPSHOT.jar"/>
 	<classpathentry kind="output" path="target/classes"/>
```

Comment:
1.2 does not contain the just open sourced `SnippetFormatter` class: https://github.com/google/google-java-format/commit/4ae067aef19d0882c5a61f514ea40fa88e9b7067